web_diagram_position/static/src/js/diagram_controller.js

```diff
@@ -0,0 +1,208 @@
+                    if (widget.$dropdown) widget.$dropdown.unbind();
+                }
+            });
+
+        });
+        pop.on('closed', this, this.reload.bind(this));
+    },
+    /**
+     * Custom event handler that opens a popup to edit an edge given its id
+     *
+     * @private
+     * @param {OdooEvent} event
+     */
+    _onEditEdge: function (event) {
+        var state = this.model.get();
+        let pop = new dialogs.FormViewDialog(this, {
+            res_model: state.connector_model,
+            res_id: parseInt(event.data.id, 10),
+            context: this.context,
+            title: _.str.sprintf("%s %s", _t("Open:"), _t('Transition')),
+            on_saved: this.reload.bind(this),
+        }).open();
+
+        pop.opened().then(function(){
+            let fields_to_disable = [state.parent_field];
+            let renderer = pop.form_view.renderer;
+            _.each(renderer.allFieldWidgets[renderer.state.id], function (widget) {
+                if (fields_to_disable.includes(widget.name)) {
+                    if (widget.$input)  widget.$input.prop('disabled', true);
+                    if (widget.$dropdown) widget.$dropdown.unbind();
+                }
+            });
+        });
+    },
+    /**
+     * Custom event handler that opens a popup to edit the content of a node
+     * given its id
+     *
+     * @private
+     * @param {OdooEvent} event
+     */
+    _onEditNode: function (event) {
+        var state = this.model.get();
+        let pop = new dialogs.FormViewDialog(this, {
+            res_model: state.node_model,
+            res_id: event.data.id,
+            context: this.context,
+            title: _.str.sprintf("%s %s", _t("Open:"), _t('Activity')),
+            on_saved: this.reload.bind(this),
+        }).open();
+
+        pop.opened().then(function(){
+            let fields_to_disable = [state.parent_field];
+            let renderer = pop.form_view.renderer;
+            _.each(renderer.allFieldWidgets[renderer.state.id], function (widget) {
+                if (fields_to_disable.includes(widget.name)) {
+                    if (widget.$input)  widget.$input.prop('disabled', true);
+                    if (widget.$dropdown) widget.$dropdown.unbind();
+                }
+            });
+        });
+    },
+    /**
+     * Custom event handler that removes an edge given its id
+     *
+     * @private
+     * @param {OdooEvent} event
+     */
+    _onRemoveEdge: function (event) {
+        var self = this;
+        Dialog.confirm(this, (_t("Are you sure you want to remove this transition?")), {
+            confirm_callback: function () {
+                var state = self.model.get();
+                self._rpc({
+                        model: state.connector_model,
+                        method: 'unlink',
+                        args: [event.data.id],
+                    })
+                    .then(self.reload.bind(self));
+            },
+        });
+    },
```

Comment:
I don't see any change in this method compared with original implementation. You can remove this method.